src/popup/Components/PopupItem.svelte

```diff
@@ -27,12 +27,12 @@
     }
 
     @media (hover: hover) {
-        .item:hover {
+        .item.self:hover {
```

Comment:
```suggestion
        :global(:is(.even, .odd)) .item:hover {
```

The reason I believe this is happening is because of [this line of style code](https://github.com/quoid/userscripts/blob/5f513191c4a61baecce4564ea843401178a73678/src/popup/App.svelte#L704). It is a `:global` and has more specify than `item:hover`.

Rather than adding a new class, I think we should just address the specificity issue with some like the above, or by using an `!important` tag. Here are the `:is` docs: https://developer.mozilla.org/en-US/docs/Web/CSS/:is

I think `:global` was probably not the best idea to begin with, but it was a quick solution. I now would prefer to rather pass a prop to `PopupItem` to determine what the non-hovered background color should be. We can address this later, by adding a comment, something like this:

```css
/* @todo: refactor this global style rule, will need to address style assignment in App.svelte as well */
.item:hover,
:global(:is(.even, .odd)) .item:hover {
    background-color: rgb(255 255 255 / 0.075);
}
```